jobs/audit_jobs.py

```diff
@@ -19,17 +19,25 @@
 from __future__ import absolute_import  # pylint: disable=import-only-modules
 from __future__ import unicode_literals  # pylint: disable=import-only-modules
 
+import itertools
+
 from jobs import base_jobs
 from jobs import job_utils
 from jobs.transforms import audits_registry
 from jobs.transforms import base_model_audits
+from jobs.types import audit_errors
 import python_utils
 
 import apache_beam as beam
 
 AUDIT_DO_FN_TYPES_BY_KIND = audits_registry.get_audit_do_fn_types_by_kind()
 KIND_BY_INDEX = tuple(AUDIT_DO_FN_TYPES_BY_KIND.keys())
 
+PROP_REFERENCES_BY_KIND = audits_registry.get_property_relationships_by_kind()
```

Comment:
Note that property won't work because it'll conflict with Python's property (that's why I used prop).

I think it's better to stick with prop because that's exactly what these are, even if we've used fields before I think it's better to stick with their actual name (StringProperty, IntegerProperty, etc.)

I don't see any ambiguity about what a prop is either, so I don't think it deserves to be changed.

Reply:
I would suggest using "property" everywhere unless you can't for some reason. Already here in this PR we are using "prop" in some places and "property" in others.

When I first saw "prop" I actually thought it actually referred to some kind of "prop" (i.e. support). Unfortunately, the prefix isn't entire unambiguous.